data/routers/action-routers.js

```diff
@@ -0,0 +1,133 @@
+const express = require("express");
+const router = express.Router();
+
+const actions = require("../helpers/actionModel");
+
+router.get("/api/actions", (req, res) => {
+  const { id } = req.body;
```

Comment:
For `get` operations, it is best practice to pass all queries through req.params and not through req.body